src/js/media-time-range.js

```diff
@@ -161,7 +162,7 @@ class MediaTimeRange extends MediaChromeRange {
 
     this._refreshBar = () => {
       const delta = (performance.now() - this._updateTimestamp) / 1000;
-      this.range.value = this.mediaCurrentTime + delta;
+      this.range.value = this.mediaCurrentTime + delta * this.mediaPlaybackRate;
```

Comment:
nit(non-blocking): Consider adding parens for clarity of order of operations (even though this is mathematically correct)